extension/lib/shopify.api.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @param {object} config
+ * @return {{}}
+ */
+module.exports = function (config) {
+  const module = {}
+  const shopDomain = 'https://' + config.shopifyShopAlias + '.myshopify.com'
+
+  module.getCollectionListingUrl = () => {
+    return shopDomain + '/admin/collection_listings.json'
+  }
+
+  module.getCollectionProductCountUrl = (collectionId) => {
+    return shopDomain + '/admin/products/count.json?collection_id=' + collectionId
+  }
+
+  module.getGraphQlUrl = () => {
+    return shopDomain + '/api/graphql'
+  }
+
+  /**
+   * Adds necessary Header-Informations which are needed for the Request
+   * @returns {object}
+   */
+  module.getAdminApiRequestHeader = () => {
+    return {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Shopify-Access-Token': config.shopifyAccessToken
+      }
+    }
+  }
+
+  /**
+   * Adds necessary Header-Informations which are needed for the Request
```

Comment:
`Informations` is not an english word. There is only `information`. `Request` should be written in lowercase